snap/snapcraft.yaml

```
@@ -30,6 +30,12 @@ apps:
       - raw-usb
       - home
       - network-observe
+  qdl-ramdump:
```

you probably want to drop `qdl-`, just `ramdump`, so the app name is `qdl.ramdump` vs `qdl.qdl-ramdump`
```suggestion
  ramdump:
```

---

right. thanks for the suggestion!